source/server/admin/admin.h

```diff
@@ -494,6 +495,7 @@ class AdminImpl : public Admin,
   const std::vector<Http::OriginalIPDetectionSharedPtr> detection_extensions_{};
   const std::vector<Http::EarlyHeaderMutationPtr> early_header_mutations_{};
   const absl::optional<std::string> scheme_{};
+  const bool scheme_match_upstream_ = true;
```

Comment:
I'm wondering whether this will break backward-compatibility, and should be false.

Reply:
Set to false.